src/main/java/org/jabref/gui/entryeditor/SciteTabViewModel.java

```diff
@@ -0,0 +1,128 @@
+package org.jabref.gui.entryeditor;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Optional;
+import java.util.concurrent.Future;
+
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+
+import org.jabref.gui.AbstractViewModel;
+import org.jabref.gui.util.BackgroundTask;
+import org.jabref.gui.util.TaskExecutor;
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.net.URLDownload;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.identifier.DOI;
+import org.jabref.preferences.PreferencesService;
+
+import kong.unirest.json.JSONObject;
+import org.tinylog.Logger;
+
+public class SciteTabViewModel extends AbstractViewModel {
+
+    // The view only has three states - In Progress, found (success) and error
+    enum Status {
+        IN_PROGRESS,
+        FOUND,
+        ERROR
+    }
+
+    private static final String BASE_URL = "https://api.scite.ai/";
+
+    private final PreferencesService preferencesService;
+    private final TaskExecutor taskExecutor;
+    private final ObjectProperty<Status> status;
+    private final StringProperty searchError;
+    private Optional<SciteTallyDTO> currentResult = Optional.empty();
+
+    private Future<?> searchTask;
+
+    public SciteTabViewModel(PreferencesService preferencesService, TaskExecutor taskExecutor) {
+        this.preferencesService = preferencesService;
+        this.taskExecutor = taskExecutor;
+        this.status = new SimpleObjectProperty<>(Status.IN_PROGRESS);
+        this.searchError = new SimpleStringProperty("");
+    }
+
+    public boolean shouldShow() {
+        return preferencesService.getEntryEditorPreferences().shouldShowSciteTab();
+    }
+
+    public void bindToEntry(BibEntry entry) {
+        // If a search is already running, cancel it
+        cancelSearch();
+
+        if (entry == null) {
+            searchError.set("Null Entry!");
```

Comment:
I think Null entry is not helpful for the user, maybe something like: No active entry
And please also localize the error messages 
https://devdocs.jabref.org/code-howtos/localization.html